src/screens/LoginScreen/__test__/loginScreen.test.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { render, fireEvent, act } from '@testing-library/react-native';
+import { Provider } from 'react-redux';
+import Config from 'react-native-config';
+import { createStore } from 'redux';
+
+import LoginScreen from '../LoginScreen';
+import { GoogleSignin } from '../../../../tests/__mocks__/@react-native-google-signin/google-signin';
+import isConnected from '../../../../tests/__mocks__/@react-native-community/netinfo';
+import reducer from '../../../reducers';
```

Comment:
Use absolute import like so:
```
import LoginScreen from 'screens/LoginScreen/LoginScreen';
import { GoogleSignin } from 'tests/__mocks__/@react-native-google-signin/google-signin';
import isConnected from 'tests/__mocks__/@react-native-community/netinfo';
import reducer from 'reducers';
```
I don't know if this works with all of the imports, but it should

Reply:
yes it does